<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="ui_mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    background-color: #F0F0F0;
    border: 1px solid #C0C0C0;
}

QTabWidget::pane QPushButton {
    background-color: #DDDDDD; 
    color: black;              
    border: 1px solid #808080; 
    border-radius: 4px;        
    padding: 6px 12px;         
    min-width: 80px;           
}


QTabWidget::pane QPushButton:hover {
    background-color: #E0E0E0; /* Hover 시 약간 밝게 */
    border-color: #606060;
}

QTabWidget::pane QPushButton:pressed {
    background-color: #C0C0C0; 
    border-color: #404040;
    padding-top: 7px;          
    padding-bottom: 5px;
}

QTabWidget::pane QPushButton:disabled {
    background-color: #EEEEEE;
    color: #A0A0A0;
    border-color: #C0C0C0;
}

QTabBar::tab {
    background-color: #E0E0E0;
    border: 1px solid #C0C0C0;
    border-bottom-color: #F0F0F0; 
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    padding: 5px 10px;
}
QTabBar::tab:selected {
    background-color: white;
    border-bottom-color: white;
}
QTabBar::tab:hover {
    background-color: #D0D0D0;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_basic">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_basic_main"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_setting">
      <attribute name="title">
       <string>Setting</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_about">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vLayout_about_main"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
